tx_check.go

```diff
@@ -27,11 +27,52 @@ func (tx *Tx) Check(options ...CheckOption) <-chan error {
 	}
 
 	ch := make(chan error)
-	go tx.check(chkConfig.kvStringer, ch)
+	go tx.check(chkConfig.kvStringer, nil, ch)
 	return ch
 }
 
-func (tx *Tx) check(kvStringer KVStringer, ch chan error) {
+func (tx *Tx) CheckPage(pageId *int, options ...CheckOption) <-chan error {
```

Comment:
@ahrtr @fuweid  i added this not to change the current cmd line 

i know its ugly :/